weblab/accounts/tests/test_signals.py

```diff
@@ -25,3 +25,20 @@ def test_email_sent_on_user_creation(superuser):
     assert 'Test User' in body
     assert 'test@example.com' in body
     assert 'http://127.0.0.1:8000/admin/accounts/user/{}/change/'.format(user.pk) in body
+
+
+@pytest.mark.django_db
+def test_delete_user_directory(client, model_with_version):
+    model = model_with_version
+    user = model.author
+
+    user_directory_repo = user.get_storage_dir('repo')
+
+    assert user_directory_repo.is_dir()
+
+    response = client.post(
+        '/accounts/%d/delete/' % user.pk,
+    )
```

Comment:
@jonc125 For some reason my call here doesn't either delete the user or their folders. In the gui the user is definitely deleted?